news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,70 @@
+
+Within the first few minutes of the medal releasing, players immediately noticed both hints and especially the medal icon referenced theyaremanycolors. Ever since the beginning of this medal's release, medal hunters knew the medal involved that map. From there, medal hunters were very quick to piece together that the hues from medal hint/flair are hinting at RGB to some capacity, but nobody knew in what way, and players were less focused on the subhint which ended up being the final crucial puzzle piece. From there, medal hunters set out to find how to input various RGB values in game, but nothing seemed to be working. Red, green, and blue shows up in the video of the map, so was it dying or receiving combo up to those sections chronologically? Could it have been to do with setting certain grades back-to-back? In the default skin, D ranks are red, A ranks are green, and B ranks are blue. Could it have been to do with mods? In the default skin, Hard Rock is red, Easy is green, but wait... Expert Hush-Hush means absolutely no difficulty reducing mods, so that couldn't be it. And while "hues of madness" seemed to be critical in figuring out the medal solution, just what in the world could "Bask in the shade of an ailing mind" mean? Medal hunters thought all of this within the first 2 days. Medal hunters were on the right track so much, that Ephemeral declared that medal hunters would not need a single hint from them, however, with no way to tell if guesses were getting closer or not to the solution, insanity only grew bigger.
+
+The community was trying their best to *hunt* for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. *On this journey of basking in the shade of an ailing mind*, the community had received various hints. The first notable one was from Ephemeral mentioning that this medal could involve "ARG" (alternate reality game) which blasted medal hunters' sanity away even more. The possibilities seemed endless. Another hint was a video recorded by [Glassive](https://osu.ppy.sh/users/12453848), one of medal hunters at [cavoe's osu! event (COE)](/wiki/Community/cavoe's_osu!_event), featuring [peppy](https://osu.ppy.sh/users/2) himself!
+
+<div align="center">
+  <video width="50%" controls>
+    <source src="https://assets.ppy.sh/media/news/coe_video_with_peppy.mp4?20230902" type="video/mp4" preload="none">
+  </video>
+</div>
+
+peppy said the solution may need to be altered because it depends on something that could be changed in the future. The only two possibilities were lazer and site. For those that guessed site and thought difficulty colors, they used the hint to their fullest of potentials. There was a downside to this hint though. While peppy basically confirmed this theory about diff's colors being correct, medal hunters also got a little mislead.
```

Comment:
new sentence therefore capital letter, regardless of his name being lowercase.... at least I think so
also some rewording I thought fit better, subjective though

```suggestion
Peppy said the solution may need to be altered because it depends on something that could be changed in the future. The only two possibilities were osu!(lazer) and the website. For those that guessed website and thought difficulty colors, they used the hint to it's fullest extent. There was a downside to this hint though. While peppy basically confirmed this theory about difficulty colors being correct, medal hunters also got a little mislead.
```

Reply:
will reword it to keep peppy nickname as it should me